sdk/communication/azure-communication-sms/README.md

```diff
@@ -21,102 +21,125 @@ Azure Communication SMS is used to send simple text messages.
 
-<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L50-L60 -->
+<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L68-L74 -->
 ```java
-//Send an sms to only one phone number
-SmsSendOptions options = new SmsSendOptions();
-options.setDeliveryReportEnabled(true);
-options.setTag("Tag"); /* Optional */
-// Send the message to a list of  phone Numbers and check the response for a messages ids
-SmsSendResult response = smsClient.send(
+SmsSendResult sendResult = smsClient.send(
     "<from-phone-number>",
     "<to-phone-number>",
-    "your message",
-    options /* Optional */);
-System.out.println("MessageId: " + response.getMessageId());
+    "Hi");
+
+System.out.println("Message Id: " + sendResult.getMessageId());
+System.out.println("Send Result Successful:" + sendResult.isSuccessful());
 ```
-### Sending a message to multiple recipients
-Use the `send` function to send a new message to a list of phone numbers.
-Once you send the message, you'll receive a PagedIterable response where you can access several
-properties such as the message id with the `messageResponseItem.getMessageId()` function.
+### Send a 1:N SMS Message
+To send a SMS message to a list of recipients, call the `send` or `sendWithResponse` function with a list of recipient phone numbers. You may also add pass in an options object to specify whether the delivery report should be enabled and set custom tags.
 
-<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L64-L76 -->
+<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L80-L94 -->
 ```java
-//Send an sms to multiple phone numbers
 SmsSendOptions options = new SmsSendOptions();
 options.setDeliveryReportEnabled(true);
-options.setTag("Tag"); /* Optional */
-// Send the message to a list of  phone Numbers and check the response for a messages ids
-Iterable<SmsSendResult> responseMultiplePhones = smsClient.send(
+options.setTag("Tag");
+
+Iterable<SmsSendResult> sendResults = smsClient.sendWithResponse(
     "<from-phone-number>",
-    new ArrayList<String>(Arrays.asList("<to-phone-number1>", "<to-phone-number2>")),
-    "your message",
+    Arrays.asList("<to-phone-number1>", "<to-phone-number2>"),
+    "Hi",
     options /* Optional */,
-    null);
-for (SmsSendResult messageResponseItem : responseMultiplePhones) {
-    System.out.println("MessageId sent to " + messageResponseItem.getTo() + ": " + messageResponseItem.getMessageId());
+    Context.NONE).getValue();
+
+for (SmsSendResult result : sendResults) {
+    System.out.println("Message Id: " + result.getMessageId());
+    System.out.println("Send Result Successful:" + result.isSuccessful());
+}
 ```
 
 
 ## Troubleshooting
 
-In progress.
+All SMS service operations will throw an exception on failure.
```

Comment:
I do not think we throw on a batch sms send. Not even sure for the singular one. Do we expect them to check the status of the result for single sms like batch?  Maybe we should add an example for that in the future too.

Reply:
Yes, I was just on a call with the SMS team on this. We will be adding more details to the troubleshooting section to reflect this.